server/models/userListModel.js

```diff
@@ -13,6 +13,14 @@ const UserListModel = db.define("UserList", {
     allowNull: false,
     unique: "uniqueTitleForUser",
   },
+  imageUrl: {
+    type: dt.STRING(128),
+    allowNull: true,
+  },
+  items: {
+    type: dt.INTEGER,
+    defaultValue: 0,
```

Comment:
Is items nullable by intention?

Reply:
this was to set the initial counter value to 0. not a good idea?

Reply:
The defaultValue property is fine, it's just that I think numItems should also have an allowNull property set to false.
If it's fine by you, I'll probably go ahead and add it in when I'm correcting the column names.